### PR TITLE
Stlye updates

### PR DESCRIPTION
**Pull request by johnhof**
@ptariche @lan-nguyen91

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1889,359 +1889,3 @@ Other Style Guides
     ```
 
 **[⬆ back to top](#table-of-contents)**
-
```
Comment (johnhof) [inline README.md:1892]: Just removed gross Client side stuff

File: README.md [hunk 2]
```diff
@@ -1356,7 +1356,7 @@ Other Style Guides
     });
     ```
 
-  - [18.3](#18.3) <a name='18.3'></a> Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). Place no space before the argument list in function calls and declarations.
+  - [18.3](#18.3) <a name='18.3'></a> Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). As well as a space before the argument list in function calls and declarations.
```
Comment (johnhof) [inline README.md:1359]: Only real change from `function()` to `function ()`

I can go either way on this, but I know sublime doesnt highlight `function()` as an actual function. Or at least it didnt when I used it

**Comment by ptariche** [comment]
++

File: README.md (hunk 2 above)
Comment (lan-nguyen91) [inline README.md:1359]: Just to be clear, function with name stay :
function test() , and not function test ()

Comment (johnhof) [inline README.md:1359]: I prefer `function test ()` personally. what are your thoughts?

**Comment by johnhof** [comment]
IT IS DECIDED

https://www.youtube.com/watch?v=YZJIn7u9Cm4

**Comment by lan-nguyen91** [comment]
Too many rules for beginner coder like us x.x

**Pull request merged**